app/handler/commands.js

```diff
@@ -657,6 +658,28 @@ exports.handleMessage = function (steamID, message) {
                 client.chatMessage(steamID, 'Removed "' + entry.name + '".');
             }
         });
+    } else if (isAdmin && command === 'trades') {
+        const dateNow = new Date().getTime(); // Gets date & time in milliseconds
+        const offerData = manager.pollData.offerData
+
+        let tradeToday = 0;
+        let tradeTotal = 0;
+        for (const offerID in offerData) {
+            if (!Object.prototype.hasOwnProperty.call(offerData, offerID)) {
+                continue;
+            }
+
+            if (offerData[offerID].handledByUs === true && offerData[offerID].isAccepted === true) {
+                // Sucessful trades handled by the bot
+                tradeTotal++;
+
+                if (offerData[offerID].finishTimestamp >= (dateNow - 86400000)) {
```

Comment:
Make day start and end so we get accurate data to current day.

Reply:
It is better to just do it over the last 24 hours, it is easier for people to read and it does not change depending on the time off the day.